test/functional/feature_dusd_loans.py

```diff
@@ -135,10 +135,10 @@ def goto_gce_height(self):
 
     def goto_next_height(self):
         blockHeight = self.nodes[0].getblockcount()
-        if self.nextnetworkupgradeheight > blockHeight:
-            self.nodes[0].generate((self.nextnetworkupgradeheight - blockHeight) + 2)
+        if self.metachainheight > blockHeight:
+            self.nodes[0].generate((self.metachainheight - blockHeight) + 2)
         blockchainInfo = self.nodes[0].getblockchaininfo()
-        assert_equal(blockchainInfo["softforks"]["nextnetworkupgrade"]["active"], True)
+        assert_equal(blockchainInfo["softforks"]["Metachain"]["active"], True)
```

Comment:
Case again

Reply:
Fixed in 8b51126bc0a87b835a5209b4a197ca65fc61ebc5

Reply:
Fixed: f0c313f50bfcc86eaea109bb0f1ec4205f01cb97